Sources/Player/PlaybackEngine/Internal/PlayerItemLoader.swift

```diff
@@ -40,8 +44,7 @@ private extension PlayerItemLoader {
 			return try await (metadata(of: storedMediaProduct), playerLoader.load(storedMediaProduct))
 		}
 
-		if PlayerWorld.developmentFeatureFlagProvider.isOffliningEnabled,
-		   let offlineEntry = try? offlineStorage?.get(key: mediaProduct.productId),
+		if let offlineEntry = try? offlineStorage?.get(key: mediaProduct.productId),
```

Comment:
I think we don't need to control the flag here because storage will be nil when the flag is off.

Reply:
Do we need to worry about scenario where user had the flag on (and had storage) and while using the app, it changed to off?

Reply:
I finally addressed this. Makes it a bit more complex, but I think it is worth it to avoid weird situations. Once the Flag is removed we will be able to simplify it.

Reply:
nice, thanks!